parts/kuberneteswindowssetup.ps1

```diff
@@ -61,11 +58,25 @@ $global:SecurityGroupName = "{{WrapAsVariable "nsgName"}}"
 $global:VNetName = "{{WrapAsVariable "virtualNetworkName"}}"
 $global:RouteTableName = "{{WrapAsVariable "routeTableName"}}"
 $global:PrimaryAvailabilitySetName = "{{WrapAsVariable "primaryAvailabilitySetName"}}"
-$global:NeedPatchWinNAT = $false
-
+$global:NetworkPolicy = "{{WrapAsVariable "networkPolicy"}}"
+$global:VnetIntegration = "{{WrapAsVariable "vnetIntegration"}}"
+$global:VNetCNIPluginsURL = "{{WrapAsVariable "vnetCniWindowsPluginsURL"}}"

```

Comment:
What is the default value of this URL and  how is the plugin zip uploaded/tested?

Reply:
The default value is specified in:
https://github.com/Azure/acs-engine/blob/8949e3f22a5b2720f4970dba6121ac700c8d51f8/pkg/acsengine/defaults.go#L21

It is up to you to decide which version to upload to your ACS mirror. I believe Dmitry has some test coverage, in addition to the unit tests in the CNI plugin repo.